include/geode/model/mixin/core/detail/components_storage.h

```diff
@@ -153,27 +153,6 @@ namespace geode
                 return mapping;
             }
 
-        protected:
```

Comment:
This is kind of a breaking change but it will only affect deep code and it is in an advanced header!